Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/TimeAndDateExtensions.kt

```diff
@@ -80,5 +80,7 @@ object TimeAndDateExtensions {
 
     fun LocalDate.ageInDays(now: LocalDate) = Days.daysBetween(this, now).days
 
-    fun Instant.toLocalDate() = this.toDateTime(DateTimeZone.UTC).toLocalDate()
+    fun Instant.toLocalDate(timezone: DateTimeZone = DateTimeZone.UTC) = this.toDateTime(timezone).toLocalDate()
```

Comment:
This is for sure an improvement.

I'm confused though, wouldn't one use the local timezone as default for a local date?

Reply:
I agree that it's a bit confusing, the actual time classes are called `LocalTime` and `LocalDate`, internally we always work with `UTC` time. We need `LocalTime` and `LocalDate` for the calculations though (especially formatting). I'm open for suggestions if you have an idea for a better name.

Reply:
toLocaleDateFormat ?

Reply:
I see why you use the UTC date as it is independent from timezone changes. Maybe toTimeZoneAdjustedLocalDate. However it is only minor thing, especially as it has been there before.

Reply:
I agree that it should probably be named something like "toUTCLocalDate" or something similar, but closing this for now as I noticed we also use this line of code for the risk calculation which is currently being changed on `release/1.8.x` I want to avoid unnecessary conflicts there.